### ISSUE TITLE
Undefined constant 'GLOB_BRACE'

### ISSUE THREAD

**Issue by maks-rafalko** [weirdan/doctrine-psalm-plugin#52]
Docker image:

```
php:7.4.3-fpm-alpine
```

#### How to reproduce

```
composer require --dev weirdan/doctrine-psalm-plugin
vendor/bin/psalm-plugin enable weirdan/doctrine-psalm-plugin

vendor/bin/psalm
```

Stacktrace:

```
ncaught Error: Undefined constant 'GLOB_BRACE' in /srv/api/vendor/weirdan/doctrine-psalm-plugin/Plugin.php:32
Stack trace:
#0 /srv/api/vendor/weirdan/doctrine-psalm-plugin/Plugin.php(22): Weirdan\DoctrinePsalmPlugin\Plugin->getStubFiles()
#1 /srv/api/vendor/vimeo/psalm/src/Psalm/Config.php(1161): Weirdan\DoctrinePsalmPlugin\Plugin->__invoke(Object(Psalm\PluginRegistrationSocket), NULL)
#2 /srv/api/vendor/vimeo/psalm/src/Psalm/Internal/Analyzer/ProjectAnalyzer.php(493): Psalm\Config->initializePlugins(Object(Psalm\Internal\Analyzer\ProjectAnalyzer))
#3 /srv/api/vendor/vimeo/psalm/src/psalm.php(594): Psalm\Internal\Analyzer\ProjectAnalyzer->check('/srv/api/', false)
#4 /srv/api/vendor/vimeo/psalm/psalm(2): require_once('/srv/api/vendor...')
#5 {main}

```

**Comment by weirdan** [comment]
That's fixed in #50. I'll carve a new release shortly.

**Comment by weirdan** [comment]
Fix released in [0.10.0](https://github.com/weirdan/doctrine-psalm-plugin/releases/tag/0.10.0).

**Comment by maks-rafalko** [comment]
Thank you!
